pyproject.toml

```diff
@@ -1,8 +1,45 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+
 [tool.black]
 line-length = 79
+skip-string-normalization = true
+py36 = false
 include = '\.pyi?$'
 exclude = '''
-/(
-    \.git
-)/
+(
+  /(
+      \.git
+    | \.env
+    | \.tox
+    | build
+    | deform\.egg-info
+    | dist
+    | docs
+    | env
+    | [Ff]irefox*
+  )/
+)
 '''
+
+# This next section only exists for people that have their editors
+# automatically call isort. Black already sorts entries on its own when run.
```

Comment:
Black doesn't sort entries at all.

Reply:
Missed this before merging. I pulled it from [Pyramid](https://github.com/Pylons/pyramid/pull/3604/files#diff-522adf759addbd3b193c74ca85243f7dR21-R22). Should that be updated as well?

Reply:
Yeah, I didn't realise it was in the Pyramid one.